.gitignore

```diff
@@ -8,7 +8,10 @@ build/
 *.egg-info/
 
 # Django
-migrations/
+migrations/*
+!migrations/__init__.py
+ms_game_chat/backend/backend_app/migrations/0001_initial.py
```

Comment:
please check if with this gitignore the migrations folder still exist and includes only the init file